superset-frontend/src/setup/setupColors.js

```diff
@@ -16,19 +16,19 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import airbnb from '@superset-ui/color/esm/colorSchemes/categorical/airbnb';
-import categoricalD3 from '@superset-ui/color/esm/colorSchemes/categorical/d3';
-import echarts from '@superset-ui/color/esm/colorSchemes/categorical/echarts';
-import google from '@superset-ui/color/esm/colorSchemes/categorical/google';
-import lyft from '@superset-ui/color/esm/colorSchemes/categorical/lyft';
-import preset from '@superset-ui/color/esm/colorSchemes/categorical/preset';
-import sequentialCommon from '@superset-ui/color/esm/colorSchemes/sequential/common';
-import sequentialD3 from '@superset-ui/color/esm/colorSchemes/sequential/d3';
+import airbnb from '@superset-ui/core/esm/color/colorSchemes/categorical/airbnb';
+import categoricalD3 from '@superset-ui/core/esm/color/colorSchemes/categorical/d3';
+import echarts from '@superset-ui/core/esm/color/colorSchemes/categorical/echarts';
+import google from '@superset-ui/core/esm/color/colorSchemes/categorical/google';
+import lyft from '@superset-ui/core/esm/color/colorSchemes/categorical/lyft';
+import preset from '@superset-ui/core/esm/color/colorSchemes/categorical/preset';
+import sequentialCommon from '@superset-ui/core/esm/color/colorSchemes/sequential/common';
+import sequentialD3 from '@superset-ui/core/esm/color/colorSchemes/sequential/d3';
```

Comment:
Not that this behavior is being changed, but is there some reason these have to be pulled in from `/esm/` and aren't exported like the other components in `superset-ui/core`? It feels like these should be exported as `categoricalColorSchemes.*` and `sequentialColorSchemes.*` or similar.

Reply:
I don't feel like doing additional refactor in this already humongous PR. For this specific case, I almost feel these colors should probably be move out of the `@superset-ui/core` package and placed under `incubator-superset` instead.

Reply:
Yeah, moving these to `incubator-superset` probably makes more sense. Also not recommending changing this now, this was mostly a question and proposal for future changes post merge.

Reply:
we could also add the export from index and get rid of deep import later.
Agree not doing it in this humongous PR